systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/BaseClients.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.kafkaclients.internalClients;
+
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.sundr.builder.annotations.Buildable;
+
+import java.security.InvalidParameterException;
+
+@Buildable(editableEnabled = false)
+public class BaseClients {
```

Comment:
Should this be `abstract`?

Reply:
changed to `abstract`, makes sense, thanks :)